### PR TITLE
fix(checkbox): make a solid background color

### PR DESCRIPTION
**Pull request by LucyChyzhova**
fix: https://github.com/Lundalogik/crm-feature/issues/2612
## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/checkbox/checkbox.scss [hunk 1]
```diff
@@ -14,10 +14,21 @@
...
+                    }
+                }
+            }
+        }
+
         &.mdc-checkbox--invalid {
             .mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate)
                 ~ .mdc-checkbox__background {
-                border-color: var(--lime-error-text-color);
+                border-color: var(--lime-red-text-color);
```
Comment (adrianschmidt) [inline src/components/checkbox/checkbox.scss:31]: Hmm… isn't "error" the one we should use for invalid stuff? Or rather, maybe we should create a separate variable that's called something with "invalid". Generally, we want to avoid using the names of colours in variables like this, since, if we one day decide that we want to use a different colour for invalid, say purple for example, it would make no sense for it to be called `--lime-red-text-color`.

Comment (adrianschmidt) [inline src/components/checkbox/checkbox.scss:31]: Also, this change doesn't seem to be related to the background color change? Should it perhaps be in a separate commit?

Comment (Kiarokh) [reply src/components/checkbox/checkbox.scss:31]: this must have happened by accident  :)

Comment (LucyChyzhova) [reply src/components/checkbox/checkbox.scss]: yes, you are right, I have changed it accidentally by testing something... fixed now

**Review by Kiarokh — APPROVED** [review]

File: src/components/checkbox/checkbox.scss [hunk 2]
```diff
@@ -35,16 +46,15 @@
...
+    }
 
     label {
         cursor: pointer;
         line-height: normal;
         padding-left: 0;
 
-        &.mdc-checkbox--disabled {
-            opacity: 0.37;
-        }
```
Comment (adrianschmidt) [inline src/components/checkbox/checkbox.scss:46]: Was this change intentional? 😊

Comment (LucyChyzhova) [reply src/components/checkbox/checkbox.scss:46]: yes :) 0.37 was too transparent
